platform/platform.hpp

```diff
@@ -293,6 +294,13 @@ class Platform
   static EConnectionType ConnectionStatus();
   static bool IsConnected() { return ConnectionStatus() != EConnectionType::CONNECTION_NONE; }
 
+  /// Uploads |filePath| to the |url| over https.
```

Comment:
Please add a comment describing that the file must be deleted in implementation after successful upload

Reply:
Done